lib/l10n/app_fi.arb

```diff
@@ -29,7 +29,7 @@
   "invoice_btc_address_network_error": "Bitcoin-osoitteen haku palvelimelta epäonnistui.\nTarkista Internet-yhteyden toiminta.",
   "invoice_btc_address_action_retry": "YRITÄ UUDELLEEN",
   "invoice_btc_address_action_close": "SULJE",
-  "invoice_btc_address_warning_with_min_fee_account_connected": "Osoitteeseen voi lähettää vähintään {minSats} ja enintään {maxSats}. Yli {liquidity} lähetyksestä veloitetaan {setUpFee}% avauskulu ja vähintään {minFee}.",
+  "invoice_btc_address_warning_with_min_fee_account_connected": "Osoite vastaanottaa\nvähintään {minSats} ja enintään {maxSats} maksuja.\nYli {liquidity} vastaanotosta veloitetaan\n{setUpFee}% avauskulu ja kuitenkin vähintään {minFee}.",
```

Comment:
@tlindi this `\n` adds a new line, I think you should avoid it unless you are absolutely sure it works properly in different screen sizes.